doc/api/cli.md

```diff
@@ -1289,6 +1289,27 @@ added: v18.0.0
 Configures the test runner to only execute top level tests that have the `only`
 option set.
 
+### `--test-shard`
+
+<!-- YAML
+added: REPLACEME
+-->
```

Comment:
should we append here that it's already available in node 20 (it was backported)?

Reply:
No